test/conftest.py

```diff
@@ -1,10 +1,27 @@
 import pytest
 import torch
+from multiprocessing import Process
 
 import syft
 from syft import TorchHook
 
 
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
This should not be auto used and scope should be function. (Just don't pass parameter s)